### ISSUE TITLE
User hamburger menu has insufficient contrast

### ISSUE THREAD

**Issue by Woebin** [JohnXLivingston/peertube-plugin-livechat#722]
**Describe the bug**
This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 1.4.11 (AA) Non-text contrast](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast)

People with reduced vision often have difficulty seeing components and graphics in the interface that do not have high enough contrasts against the background. This includes buttons, form fields, icons, illustrations and diagrams (such as curves and arrows).

You have a hamburger menu button on chat messages that appears on hover or focus:
![Hamburger menu.](https://github.com/user-attachments/assets/ab773b5d-257b-484e-8312-d756a8178cb8)
This button has insufficient color contrast against the background.

**To Reproduce**
1. Hover with the mouse pointer over a message in chat.
2. See the hamburger menu button appear. It has insufficient color contrast against the background.

**Expected behavior**
The button should have enough contrast to clearly stand out against the background, which means a ratio of 3:1 or more. You can measure the contrast using a variety of tools, for example the one at [](https://contrast.report).

**Server:**
 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: 14.0.0

**Desktop:**
 - OS: MacOS 15.5
 - Browser: Google Chrome Version 137.0.7151.120 (Official Build) (arm64)

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).
